src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -273,10 +273,10 @@ public int size() {
      *
      * @Deprecated use {@link Strings#repeat} directly. Left here temporarily for the ease of porting GATK3 code.
      */
-    @Deprecated
+    /* this method is useless so I have removed it
     public static String dupString(final String s, final int nCopies) {
         return Strings.repeat(s, nCopies);
-    }
+    }*/
```

Comment:
Just delete this whole block of code.  We can always bring it back from history if we need it.  Be sure to remove the comment attached to it as well.